stubs/third_party_taint/boto3_clients_sources_sinks.pysa

```diff
@@ -22,3 +22,6 @@ def mypy_boto3_application_insights.client.ApplicationInsightsClient.list_log_pa
 def mypy_boto3_application_insights.client.ApplicationInsightsClient.list_log_patterns() -> TaintSource[ExceptionMessage]: ...
 def mypy_boto3_application_insights.client.ApplicationInsightsClient.list_problems() -> TaintSource[ExceptionMessage]: ...
 
+# AWS Lambda RemoteCodeExecution Sinks
+def mypy_boto3_lambda.client.LambdaClient.create_function(Code: TaintSink[RemoteCodeExecution], ImageConfig: TaintSink[RemoteCodeExecution]): ...
+def mypy_boto3_lambda.client.LambdaClient.update_function_code(ZipFile: TaintSink[RemoteCodeExecution], S3Bucket: TaintSink[RemoteCodeExecution], S3Key: TaintSink[RemoteCodeExecution], S3ObjectVersion: TaintSink[RemoteCodeExecution], ImageUri: TaintSink[RemoteCodeExecution]): ...
```

Comment:
@abishekvashok  why S3Key and S3ObjectVersion are sink for RCE? :)

Reply:
Ah... it's just me messing up 🤦  sorry. Reverted :)